### PR TITLE
Fix button size on HPO panel

### LINKED ISSUE #4535: Fix size of buttons on HPO panel
image

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Fix #4535 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Go to a case and assign one or more HPO terms
2. Create a dynamic HPO gene panel and see how it looks like on main and this branch

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
After the fix, case with no chanjo2 data: https://scout-stage.scilifelab.se/cust000/SWEDAC-2019-HG-05-Trio-NovaSeq

image

File: scout/server/blueprints/cases/templates/cases/gene_panel.html [hunk 1]
```diff
@@ -229,12 +227,15 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
           </div>
         </div>
         {% endif %} <!-- End of if case.chanjo_coverage or case.chanjo2_coverage -->
-      </div> <!-- End of Show variants in dynamic gene list -->
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/gene_panel.html:232]: Moved this line under, to line 238

File: scout/server/blueprints/cases/templates/cases/gene_panel.html [hunk 2]
```diff
@@ -130,8 +130,8 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
         </div>
       </div>
 
-      <div class="row mt-2"> <!-- Show variants in dynamic gene list -->
-        <div class="row">
+      <div>
+        <div class="row"> <!-- Show variants in dynamic gene list -->
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/gene_panel.html:134]: 👍

**Review by dnil — APPROVED** [review]
💯

**Pull request merged**